src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -277,6 +277,7 @@ def load_command_table(self, _):
         g.custom_command('list-instance-public-ips', 'list_vmss_instance_public_ips')
         g.command('list-skus', 'list_skus')
         g.custom_command('reimage', 'reimage_vmss', supports_no_wait=True, min_api='2017-03-30')
+        g.command('perform-maintenance', 'perform_maintenance', min_api='2017-12-01')
```

Comment:
Is there a help description on the command?

Reply:
yes, the auto-command will load from the sdk, which looks good to me: `Perform maintenance on one or more virtual machines in a VM scaleset.`...